Add map for overall scores

finnwoelm-sdsn opened the pull request:
Add /map for overall scores.

It's basically just a copy from ESDR 2020.

It does **not** have improved navigation, timeseries for overall score, timeslider, etc... We will add those features in separate, future pull requests.

Note: `IndicatorTimeseries` component is not added (marked as `TODO`) because it was not needed for map with overall scores. Will add that component when implementing goal and indicator maps.

finnwoelm-sdsn commented:
@Sakala85 & @DonTouzi - there are a lot of files added in this PR. Don't bother checking every single one (it will take forever).

My suggestion would be to check the following:
- [deploy preview](https://deploy-preview-12--sdr-twenty-one.netlify.app/): Are there any issues with the /map page?
- pages/map.js: Does the code for this page look good?
- Cypress tests: Did Finn test all features? Does any other feature need to be tested?

DonTouzi reviewed (changes requested):
Hi @finnwoelm-sdsn, for me, everything is good, very good work :)
I have some remarks: 
In the legend, is it possible to change from (Information unavailable to Unavailable information), 
Reduce the incoming zoom if possible, and 
For the map, where there is the link to unsdsn.org and the note on country boundaries (can we change in future PR), 

What do you think ?

finnwoelm-sdsn commented:
> 
> 
> Hi @finnwoelm-sdsn, for me, everything is good, very good work :)
> I have some remarks:
> In the legend, is it possible to change from (Information unavailable to Unavailable information),
> Reduce the incoming zoom if possible, and
> For the map, where there is the link to unsdsn.org and the note on country boundaries (can we change it in future PR),
> 
> What do you think ?

Thanks, @DonTouzi!! :)

1. On legend, I think we keep "Information unavailable" for alignment with the country profiles in the report:
![image](https://user-images.githubusercontent.com/55237550/119664345-893cba00-be33-11eb-8561-00291a296683.png)
1. Added Trello card for map zoom reduction on desktop devices: https://trello.com/c/IDj46EqO/66-consider-reducing-max-zoom-on-desktop-devices
2. Added Trello card for adding modal for displaying note on country boundaries: https://trello.com/c/5fmwHjqG/67-display-note-on-country-boundaries-in-modal-rather-than-alert-box

We already discussed these, so I'll merge now! 🚀
